packages/parser/lib/postprocessor/postprocessor.js

```diff
@@ -9,7 +9,8 @@ const {
 } = require("./utils");
 
 module.exports = class PostProcessor {
-  constructor() {
+  constructor(code) {
+    this.hasCRLF = code.indexOf("\r") !== -1;
```

Comment:
@reinholdk Hi thanks for the fixing it. Your solutions seems works most cases. But it plus 1 for all text nodes when whole code has one `\r` even if the text node does not contain `\r`.

```
\r\n
<html>\n
   aaa\n
   aaa\n
</html>
```

I think the primary reason for this bug is [normalization newlines](https://infra.spec.whatwg.org/#normalize-newlines) process on tokenization. I think the processing not required for us .

so what do you think about removing the below lines instead of plus 1? 
https://github.com/yeonjuan/html-eslint/blob/main/packages/parser/lib/tokenizer/preprocessor.js#L111-L117

Reply:
@yeonjuan thanks for the hint. It seems to be working fine. Have reverted my initial change.